client/src/components/Navbar.jsx

```diff
@@ -25,7 +25,7 @@ function MyNavBar(props) {
         <Navbar.Collapse className="justify-content-end">
           <Navbar.Text>
             Signed in as: <a href="#login">Cristina Rodriguez</a>
```

Comment:
Not a blocker right now, just want to make sure you're aware that this is still hard-coded to Cristina.